src/store/graph.module.js

```diff
@@ -0,0 +1,51 @@
+/* jshint esversion: 6, asi: true */
```

Comment:
This is ready for websocket functionality

Reply:
The websocket will take a while. We will need to prepare a PR for the tree view as well. I think it would be simpler to have this PR just to add the Graph component with the view changes, limiting the scope. Then add other features in subsequent PR's.